dis_013_volcano_eruptions/README.md

```diff
@@ -1,9 +1,11 @@
 ## Volcano Eruptions Dataset Near Real-time Script
 This file describes the near real-time script that retrieves and processes the [Volcanoes of the World Database - Eruptions dataset](http://volcano.si.edu/search_eruption.cfm) for [display on Resource Watch](https://resourcewatch.org/data/explore/dis013-Volcanoes-CSV).
 
-{Describe how the original data came from the source.}
+This dataset was provided by the source as a XML file. This XML was first converted into a JSON and then transformed into a table so that it could be uploaded to Carto. In order to transform the data from the JSON to the data table used by Resource Watch, the following changes were made:
```

Comment:
```suggestion
This dataset was provided by the source as a XML file. This XML was first converted into a JSON and then transformed into a table so that it could be uploaded to Carto. The 'GeoLocation' was used to create the geometry shown on Resource Watch.
```